src/components/Navigation/Ui/ButtonWithTooltip.js

```diff
@@ -6,7 +6,7 @@ import { OverlayTrigger, Tooltip } from 'react-bootstrap'
 const ButtonWithTooltip = ({ children, tip }) => {
   return (
     <OverlayTrigger rootClose placement="top" overlay={<Tooltip id="tooltip">{tip}</Tooltip>}>
-      <div className="tooltipWrapper">{children}</div>
```

Comment:
This was added to fix a problem of the Tooltip, disabling it when hovering a disabled button.
It should be kept.